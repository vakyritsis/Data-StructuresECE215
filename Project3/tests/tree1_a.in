-i 61 0 72 -p 9 -p 11 -p 71 -p 23 -p 86 -p 12 -p 33 -p 20 -p 66 -p 52 -p 39 -p 89 -p 68 -p 10 -p 88 -p 37 -p 56 -p 67 -p 79 -p 54 -p 80 -p 65 -p 42 -p 84 -p 3 -p 8 -p 64 -p 87 -p 38 -p 43 -p 73 -p 63 -p 
-f 20 
-d 84 -p 87 -p 86 -p 80 -p 79 -p 73 -p 72 -p 89 -p 71 -p 67 -p 65 -p 63 -p 64 -p 66 -p 88 -p 68 -p -q
