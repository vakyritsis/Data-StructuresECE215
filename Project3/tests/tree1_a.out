
INSERTED 61

INSERTED 0

INSERTED 72

61 0 72 

INSERTED 9

61 0 9 72 

INSERTED 11

61 9 0 11 72 

INSERTED 71

61 9 0 11 72 71 

INSERTED 23

61 9 0 11 23 72 71 

INSERTED 86

61 9 0 11 23 72 71 86 

INSERTED 12

61 9 0 12 11 23 72 71 86 

INSERTED 33

61 9 0 12 11 23 33 72 71 86 

INSERTED 20

61 9 0 12 11 23 20 33 72 71 86 

INSERTED 66

61 9 0 12 11 23 20 33 72 71 66 86 

INSERTED 52

61 9 0 12 11 23 20 33 52 72 71 66 86 

INSERTED 39

61 9 0 12 11 23 20 39 33 52 72 71 66 86 

INSERTED 89

61 9 0 12 11 23 20 39 33 52 72 71 66 86 89 

INSERTED 68

61 9 0 12 11 23 20 39 33 52 72 68 66 71 86 89 

INSERTED 10

61 9 0 12 11 10 23 20 39 33 52 72 68 66 71 86 89 

INSERTED 88

61 9 0 12 11 10 23 20 39 33 52 72 68 66 71 88 86 89 

INSERTED 37

61 12 9 0 11 10 23 20 39 33 37 52 72 68 66 71 88 86 89 

INSERTED 56

61 12 9 0 11 10 23 20 39 33 37 52 56 72 68 66 71 88 86 89 

INSERTED 67

61 12 9 0 11 10 23 20 39 33 37 52 56 72 68 66 67 71 88 86 89 

INSERTED 79

61 12 9 0 11 10 23 20 39 33 37 52 56 72 68 66 67 71 88 86 79 89 

INSERTED 54

61 12 9 0 11 10 23 20 39 33 37 54 52 56 72 68 66 67 71 88 86 79 89 

INSERTED 80

61 12 9 0 11 10 23 20 39 33 37 54 52 56 72 68 66 67 71 88 80 79 86 89 

INSERTED 65

61 12 9 0 11 10 23 20 39 33 37 54 52 56 72 68 66 65 67 71 88 80 79 86 89 

INSERTED 42

61 12 9 0 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 67 71 88 80 79 86 89 

INSERTED 84

61 12 9 0 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 67 71 88 80 79 86 84 89 

INSERTED 3

61 12 9 0 3 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 67 71 88 80 79 86 84 89 

INSERTED 8

61 12 9 3 0 8 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 67 71 88 80 79 86 84 89 

INSERTED 64

61 12 9 3 0 8 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 64 67 71 88 80 79 86 84 89 

INSERTED 87

61 12 9 3 0 8 11 10 23 20 39 33 37 54 52 42 56 72 68 66 65 64 67 71 88 80 79 86 84 87 89 

INSERTED 38

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 52 42 56 72 68 66 65 64 67 71 88 80 79 86 84 87 89 

INSERTED 43

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 65 64 67 71 88 80 79 86 84 87 89 

INSERTED 73

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 65 64 67 71 88 80 79 73 86 84 87 89 

INSERTED 63

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 80 79 73 86 84 87 89 

FOUND 20

DELETED 84

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 80 79 73 86 87 89 

DELETED 87

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 80 79 73 86 89 

DELETED 86

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 79 73 80 89 

DELETED 80

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 79 73 89 

DELETED 79

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 73 89 

DELETED 73

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 72 68 66 64 63 65 67 71 88 89 

DELETED 72

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 88 68 66 64 63 65 67 71 89 

DELETED 89

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 66 64 63 65 67 88 71 

DELETED 71

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 66 64 63 65 67 88 

DELETED 67

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 66 64 63 65 88 

DELETED 65

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 64 63 66 88 

DELETED 63

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 64 66 88 

DELETED 64

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 66 88 

DELETED 66

61 12 9 3 0 8 11 10 23 20 39 37 33 38 54 43 42 52 56 68 88 

DELETED 88

23 12 9 3 0 8 11 10 20 61 39 37 33 38 54 43 42 52 56 68 

DELETED 68

23 12 9 3 0 8 11 10 20 54 39 37 33 38 43 42 52 61 56 
